roles/ec2_instance_create_delete/README.md

```diff
@@ -0,0 +1,171 @@
+# ec2_instance_create
+
+A role to create an EC2 instance in AWS.
+
+Users can specify various parameters for instance configuration, including instance type, AMI ID, key pair, tags, and VPC/subnet configuration.
+
+This role also supports the creation of optional networking resources, such as an external security group and an Elastic IP (EIP). You can choose to wait for the EC2 instance to finish booting before continuing.
+
+## Role Variables
+
+The following variables can be set in the role to customize EC2 instance creation and networking configurations:
+
+### Role Operation
+
+* **ec2_instance_create_delete_operation**: (Optional)
+  - Goal state for the instances.
+  - "O(state=present): ensures instances exist, but does not guarantee any state (e.g. running). Newly-launched instances will be run by EC2."
+  - "O(state=running): O(state=present) + ensures the instances are running."
+  - "O(state=started): O(state=running) + waits for EC2 status checks to report OK if O(wait=true)."
+  - "O(state=stopped): ensures an existing instance is stopped."
+  - "O(state=rebooted): convenience alias for O(state=stopped) immediately followed by O(state=running)."
+  - "O(state=restarted): convenience alias for O(state=stopped) immediately followed by O(state=started)."
+  - "O(state=terminated): ensures an existing instance is terminated."
+  - "O(state=absent): alias for O(state=terminated)."
+  choices are [present, terminated, running, started, stopped, restarted, rebooted, absent]
```

Comment:
The choices listed here don't match the arg spec. I think we probably want to stick with just create and delete for this use case?